src/index.html

```diff
@@ -1,13 +1,418 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>The MET</title>
+    <link
+      rel="shortcut icon"
+      href="./images/icons/the_met_favicon.jpg"
+      type="image/x-icon"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=Roboto&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Landing Page</h1>
+    <header class="header">
+      <div class="top-bar">
+        <a href="#" class="top-bar__logo"></a>
+        <div class="top-bar__icons">
+          <a
+            href="#"
+            class="icon icon--phone"
+          >
```

Comment:
Still not fixed from the previous review, you need to check and fix it everywhere
<img width="860" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/64750958/4ad611d0-ca3d-461e-abf3-defc0d88f43e">
```suggestion
          <a href="#" class="icon icon--phone">
```